boards/doc.txt

```diff
@@ -15,4 +15,56 @@
  * configuration of a controller and some external devices such as sensors or
  * radios. All aspects concerning configuration of GPIO pins, MCU clock and
  * device drivers should go into this module.
+ *
+
+Guide to board pinouts {#pinout_guide}
+=====================================
+
+The following are the steps to take to locate specific pinouts on boards.
+This is useful when connecting external hardware to RIOT supported boards.
+
+1. Check the board documentation to see if the pinout is available
+2. If the pinout is not available or up to date check the following:
+    - `boards/<BOARD>/include/periph_conf.h` for board specific
+peripherals such as uart  or spi pins
```

Comment:
I'm always unsure: capitalise abbreviations? i.e. UART and SPI?

Also there are 2 spaces after uart, one should be enough

Reply:
Thanks, done.